implementations/code_df/commit.py

```diff
@@ -40,50 +40,49 @@ def __init__(self, items, date_range=(None, None), issrccode_obj=None):
         if self.until is None:
             self.until = utils.get_date(self.df, 'until')
 
-    def _flatten_data(self, line):
+    def _flatten_data(self, item):
         """
         This method is for cleaning a raw commit fetched by Perceval.
         Since all attributes of the data are not of importance, it is
         better to just keep the ones which are required.
 
-        :param line: a raw line fetched by Perceval, present in the JSON file
-            It is a dictionary.
+        :param item: a raw item (of type dict) fetched by Perceval,
+        present in the JSON file (items object)
```

Comment:
I had indented all following lines after the `param` tag. 
Like this:
```python
:param xyz: line1
    line2
    line3..
```
Is that a bad practise? Just curious.

Reply:
It is not a bad practice, @Polaris000. This is the result of the formatter as well.